src/main/java/com/renatusnetwork/momentum/storage/mysql/TablesDB.java

```diff
@@ -942,6 +944,18 @@ private static void createUsedCommandSigns() {
         DatabaseQueries.runQuery(query);
     }
 
+    private static void createCommandSignsCommands() {
+        String query =
+                "CREATE TABLE " + DatabaseManager.COMMAND_SIGNS_COMMANDS + "(" +
+                "name VARCHAR(20) NOT NULL, " +
+                "command VARCHAR(100) DEFAULT NULL, " +
+                // primary keys
+                "PRIMARY KEY (name)" +
```

Comment:
Primary key would be: `PRIMARY KEY (name, command)` since in this case a name can have multiple commands so if primary key is name, that would return 1 or more results which isn't a primary key. So that means in order to return exactly 1 result, we need both the name and command